be/src/main/java/com/jjikmuk/sikdorak/auth/controller/dto/response/OAuthTokenResponse.java

```diff
@@ -0,0 +1,25 @@
+package com.jjikmuk.sikdorak.auth.controller.dto.response;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.ToString;
+
+@ToString
+@Getter
+public class OAuthTokenResponse {
+
+    @JsonProperty("token_type")
+    private String tokenType;
+
+    @JsonProperty("access_token")
+    private String accessToken;
+
+    @JsonProperty("expires_in")
+    private int expiresIn;
+
+    @JsonProperty("refresh_token")
+    private String refreshToken;
+
+    @JsonProperty("refresh_token_expires_in")
+    private int refreshTokenExpiresIn;
+}
```

Comment:
```suggestion

import com.fasterxml.jackson.databind.PropertyNamingStrategies;
import com.fasterxml.jackson.databind.annotation.JsonNaming;
import lombok.Getter;
import lombok.ToString;

@Getter
@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
public class OAuthTokenResponse {

    private String tokenType;

    private String accessToken;

    private int expiresIn;

    private String refreshToken;

    private int refreshTokenExpiresIn;
}
```

`@JsonProperty` 사용이 스네이크 케이스라면 JsonNaming 을 사용해 한 번에 코드를 줄일 수 있어요

Reply:
저 방법이 훨씬 깔끔하겠네요! 또 하나 알아갑니다ㅎㅎ 반영할게요~

Reply:
suggestion commit을 사용해보는것도 ㅎㅎ